commit 64cbd887967b76b1977977da603e03c6bbdfbd1d (refs/changes/39/1739/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-04-03T15:14:43-07:00 (2 years, 6 months ago)
    
    TOOLS-1738 test-system is failing: pubapi-1003.test.js
    Reviewed by: Orlando Vazquez <orlando@joyent.com>
    Approved by: Orlando Vazquez <orlando@joyent.com>

diff --git a/test/pubapi-1003.test.js b/test/pubapi-1003.test.js
index da03b90..00872e9 100644
--- a/test/pubapi-1003.test.js
+++ b/test/pubapi-1003.test.js
@@ -6,7 +6,7 @@
  */
 
 /*
- * Copyright (c) 2014, Joyent, Inc.
+ * Copyright 2017 Joyent, Inc.
  */
 
 /*
@@ -27,7 +27,7 @@ var test = helper.test;
 
 
 test('PUBAPI-1003', function (t) {
-    exec('openssl s_client -ssl3 -no_tls1 -connect '
+    exec('openssl s_client -ssl3 -connect '
             + '$(vmadm lookup -j alias=cloudapi0 | json -a nics '
             + '| json -c \'this.nic_tag==="external"\' 0.ip):443',
         function (err, stdout, stderr) {
